frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -147,6 +147,11 @@ export default {
   },
   async created() {
     await this.getDataSourcesIndex(this.model.short_name);
+    const datatype = this.validDataTypeInQuery() ? this.$route.query.datatype : Object.keys(this.dataSourcesIndex)[0];
```

Comment:
Just a thought: I would be tempted to define a variable for `Object.keys(this.dataSourcesIndex)[0]`.

Reply:
As @inghylt mentioned, this is used only once (so I don't really recall why I made this comment). Resolving this now.